lib/streamlit/state/session_state.py

```diff
@@ -147,14 +140,12 @@ def keys(self) -> Set[str]:
         return set(self.states.keys())
 
     def items(self) -> Set[Tuple[str, Any]]:
-        i = [(k, self[k]) for k in self]
-        return set(i)
+        return {(k, self[k]) for k in self}
 
     def values(self) -> Set[Any]:  # type: ignore
-        v = [self[wid] for wid in self]
-        return set(v)
+        return {self[wid] for wid in self}
 
-    def set_from_proto(self, widget_state: WidgetStateProto):
+    def set_widgets_from_proto(self, widget_state: WidgetStateProto):
```

Comment:
nit: should this function be `set_widget_from_proto` (with the function in the SessionState calling this one keeping the plural name)